### ISSUE TITLE
Can't resolve './calculators/crc32'

### ISSUE THREAD

**Issue by rgant** [alexgorbatchev/node-crc#69]
> Error: Module not found: Error: Can't resolve './calculators/crc32' in '.../node_modules/crc'

Getting this error when I try to use crc@4.0.0

My import:
```
import crc32 from 'crc/crc32';
```

My tsconfig:
```
/* To learn more about this file see: https://angular.io/config/tsconfig. */
{
  "compileOnSave": false,
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "baseUrl": "./",
    "declaration": false,
    "downlevelIteration": true,
    "experimentalDecorators": true,
    "forceConsistentCasingInFileNames": true,
    "importHelpers": true,
    "lib": [
      "es2018",
      "dom"
    ],
    "module": "es2020",
    "moduleResolution": "node",
    "noFallthroughCasesInSwitch": true,
    "noImplicitReturns": true,
    // "noUncheckedIndexedAccess": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "outDir": "./dist/out-tsc",
    "paths": {
      "@app/*": ["src/app/*"],
      "@env/*": ["src/environments/*"],
      "@testing/*": ["src/testing/*"],
      "firebase-app": ["src/firebase-app.ts"]
    },
    "resolveJsonModule": true,
    "sourceMap": true,
    "strict": true,
    "target": "es2015"
  },
  "angularCompilerOptions": {
    "enableI18nLegacyMessageIdFormat": false,
    "strictInjectionParameters": true,
    "strictInputAccessModifiers": true,
    "strictTemplates": true
  }
}
```

SImilar errors are reported when I try `import { crc32 } from 'crc';` (Which is what I did in 3.8.0) and `import crc from 'crc';`

**Comment by Zuweilen** [comment]
I encountered a similar problem

If you want to import:
`import crc32 from 'crc/crc32'`

On the first line of this file:
node_modeules/crc/crc32.js

try changing
`import crc32 from './calculators/crc32'`
to
`import crc32 from './lib/crc32'`

**Comment by alexgorbatchev** [comment]
Should be fixed via 272197b3dc933e13f8da80551944be9d37530b62
